test/Microsoft.AspNet.Mvc.ModelBinding.Test/Metadata/ModelMetadataAttributesTest.cs

```diff
@@ -0,0 +1,158 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Reflection;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding
+{
+    public class ModelMetadataAttributesTest
+    {
+        [Fact]
+        public void GetAttributesForBaseProperty_IncludeMetadataAttributes()
+        {
+            // Arrange
+            var modelType = typeof(BaseViewModel);
+            var property = modelType.GetRuntimeProperties().Where(p => p.Name == "BaseProperty").FirstOrDefault();
```

Comment:
nit: combine `Where` and `FirstOrDefault` using the `TSource FirstOrDefault<TSource>(this IEnumerable<TSource>, Func<TSource, bool>)` overload